ERROR: Attempt to use 'texture' as a variable

Thanks for this neat library!

I'm getting this error on master right now when trying to `go run` the platformer example:

```
panic: failed to create Canvas, there's a bug in the shader: error compiling vertex shader: ERROR: 0:6: Attempt to redeclare 'texture' as a variable
ERROR: 0:21: Attempt to use 'texture' as a variable


goroutine 1 [running, locked to thread]:
github.com/faiface/pixel/pixelgl.NewCanvas.func1()
        /Users/pi/code/go/src/github.com/faiface/pixel/pixelgl/canvas.go:52 +0x138
github.com/faiface/mainthread.Call.func1()
        /Users/pi/code/go/src/github.com/faiface/mainthread/mainthread.go:64 +0x26
github.com/faiface/mainthread.Run(0x4196cb0)
        /Users/pi/code/go/src/github.com/faiface/mainthread/mainthread.go:46 +0x15f
github.com/faiface/pixel/pixelgl.Run(0x4196cb0)
        /Users/pi/code/go/src/github.com/faiface/pixel/pixelgl/run.go:32 +0x5e
main.main()
        /Users/pi/code/workspace/go/pixel/examples/platformer/main.go:394 +0x2d
exit status 2

```

minimal fix PR will follow shortly.

--- a/pixelgl/canvas.go
+++ b/pixelgl/canvas.go
@@ -311,14 +311,14 @@ func (cp *canvasPicture) Draw(t pixel.TargetTriangles) {
 const (
 	canvasPosition int = iota
 	canvasColor
-	canvasTexture
+	canvasTexCoords
 	canvasIntensity
 )
 
 var canvasVertexFormat = glhf.AttrFormat{
 	canvasPosition:  {Name: "position", Type: glhf.Vec2},
 	canvasColor:     {Name: "color", Type: glhf.Vec4},
-	canvasTexture:   {Name: "texture", Type: glhf.Vec2},
+	canvasTexCoords: {Name: "texCoords", Type: glhf.Vec2},
 	canvasIntensity: {Name: "intensity", Type: glhf.Float},
 }
 
@@ -341,11 +341,11 @@ var canvasVertexShader = `
 
 in vec2 position;
 in vec4 color;
-in vec2 texture;
+in vec2 texCoords;
 in float intensity;
 
 out vec4 Color;
-out vec2 Texture;
+out vec2 TexCoords;
 out float Intensity;
 
 uniform mat3 transform;
@@ -356,7 +356,7 @@ void main() {
 	vec2 normPos = (transPos - bounds.xy) / bounds.zw * 2 - vec2(1, 1);
 	gl_Position = vec4(normPos, 0.0, 1.0);
 	Color = color;
-	Texture = texture;
+	TexCoords = texCoords;
 	Intensity = intensity;
 }
 `
@@ -365,7 +365,7 @@ var canvasFragmentShader = `
 #version 330 core
 
 in vec4 Color;
-in vec2 Texture;
+in vec2 TexCoords;
 in float Intensity;
 
 out vec4 color;
@@ -380,7 +380,7 @@ void main() {
 	} else {
 		color = vec4(0, 0, 0, 0);
 		color += (1 - Intensity) * Color;
-		vec2 t = (Texture - texBounds.xy) / texBounds.zw;
+		vec2 t = (TexCoords - texBounds.xy) / texBounds.zw;
 		color += Intensity * Color * texture(tex, t);
 		color *= colorMask;
 	}

